charts/sscs-bulk-scan/values.yaml

```
@@ -19,3 +17,16 @@ java:
     ROOT_LOGGING_LEVEL: INFO
     LOG_OUTPUT: single
     IDAM_OAUTH2_REDIRECT_URL: "https://evidence-sharing-preprod.sscs.reform.hmcts.net"
+    IDAM_URL: "https://idam-api.aat.platform.hmcts.net"
```

https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L11
```suggestion
    IDAM_URL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
```